packages/ui/src/views/repo/webhooks/webhook-executions/repo-webhook-executions-list-page.tsx

```diff
@@ -99,8 +100,9 @@ const RepoWebhookExecutionsPage: FC<RepoWebhookExecutionsPageProps> = ({
                             : 'Invalid'}
                       </Badge>
                     </TableCell>
-                    <TableCell className="text-right">
-                      <Text size={2}>{timeAgo(execution.created)}</Text>
+                    <TableCell className="text-right relative">
+                      {/* Add relative positioning */}
+                      <TimeAgoHoverCard timeStamp={execution.created ?? 0} />
```

Comment:
can we look into updating the `timeAgo()` function to return JSX instead of a string?
that way we don't need to change anything to adopt this hovercard across the app.